README.md

```diff
@@ -162,4 +184,22 @@ $ id -u $USER  # check UID
 $ id -g $USER  # check GID
 ```
 
-In Ascender, default value of both is `1000`. If UID or GID of your local PC is not `1000`, you need to modify the value of `UID` or `GID` inside of `docker-compose.yaml` to align your local PC (please edit their values from `1000`).
\ No newline at end of file
+In Ascender, default value of both is `1000`. If UID or GID of your local PC is not `1000`, you need to modify the value of `UID` or `GID` inside of `docker-compose.yaml` to align your local PC (please edit their values from `1000`).
+
+### Compatibility issue between PyTorch and Poetry
```

Comment:
This is about English. I didn't know this usage of `Compatibility`. I only knew `Compatibility` kind of `Replacable`. Thank you.